sentry-android-core/src/main/java/io/sentry/android/core/AndroidOptionsInitializer.java

```diff
@@ -305,23 +305,23 @@ private static void initializeCacheDirs(
     options.setCacheDirPath(cacheDir.getAbsolutePath());
     options.setProfilingTracesDirPath(profilingTracesDir.getAbsolutePath());
 
-    if (options.isProfilingEnabled()) {
-      profilingTracesDir.mkdirs();
-      final File[] oldTracesDirContent = profilingTracesDir.listFiles();
-
-      options
-          .getExecutorService()
-          .submit(
-              () -> {
-                if (oldTracesDirContent == null) return;
-                // Method trace files are normally deleted at the end of traces, but if that fails
-                // for some
-                // reason we try to clear any old files here.
-                for (File f : oldTracesDirContent) {
-                  FileUtils.deleteRecursively(f);
-                }
-              });
-    }
+    // We don't have the options set by code (manual initialisation) at this point, so we have no
```

Comment:
That's why we do the file handling in the Sentry init class
https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/Sentry.java#L222-L229=
Can we move this logic in there? so we don't need create a folder unnecessarily

Reply:
alright, but i should move `profilingTracesDirPath` from `SentryAndroidOptions` to `SentryOptions` (`profilingEnabled` is already in `SentryOptions`).
Is it fine?